pages/editArticle.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import withRedux from 'next-redux-wrapper';
+
+import Text from 'components/common/Text';
+import PageLayout from 'components/common/layout/PageLayout';
+import EditArticleForm from 'components/articles/edit/EditArticleForm';
+
+import initStore from 'redux/store';
+import { actions as articlesActions, selectors } from 'redux/ducks/articles';
+import { actions as auth, selectors as authSelectors } from 'redux/ducks/auth';
+import request from 'utils/request';
+import { LangType } from 'utils/customPropTypes';
+import { Router, ROUTES_NAMES } from 'routes';
+
+const mapStateToProps = (state, { url: { query } }) => ({
+  article: selectors.getCurrent(state, query.slug),
+  articleLocale: query.articleLocale || selectors.getLocaleBySlug(state, query.slug),
+  error: selectors.isError(state),
+  permissions: authSelectors.getPermissions(state),
+});
+
+class EditArticlePage extends Component {
+  static getInitialProps(ctx) {
+    const { query: { slug } } = ctx;
+    return request.populate(
+      ctx,
+      [auth.getCurrentUser, slug && articlesActions.fetchBySlug.bind(null, slug)].filter(Boolean)
```

Comment:
oh, I thought, that we call filter on this statement ``articlesActions.fetchBySlug.bind(null, slug)``, but there is ``]`` :)
could you change formatting, please?

Reply:
I see, okay